modules/installation-user-defined-tags-azure.adoc

```diff
@@ -21,51 +21,49 @@ You can use the `.platform.azure.userTags` field in the `install-config.yaml` fi
 additionalTrustBundlePolicy: Proxyonly <1>
 apiVersion: v1
 baseDomain: catchall.azure.devcluster.openshift.com <2>
-featureSet: TechPreviewNoUpgrade <3>
-compute: <4>
+compute: <3>
 - architecture: amd64
-  hyperthreading: Enabled <5>
+  hyperthreading: Enabled <4>
   name: worker
   platform: {}
   replicas: 3
-controlPlane:  <4> <6>
+controlPlane:  <3> <5>
   architecture: amd64
```

Comment:
Applies to this entire YAML: Reusing callout numbers is not permitted because it does not render correctly on the Customer Portal. Please update this so that each callout number is only used once and then re-request review. Thank you!

(I am also confused about how some of the callouts apply to their descriptions, but that might just be my unfamiliarity with the topic. :))

Reply:
Addressed